### PR TITLE
Surf s4 t2 setup db connection

### PR DESCRIPTION
**Pull request by JoshuaBennet**

### CODE REVIEW

File: src/Controllers/API/AddAptitudeScoreController.php [hunk 1]
```diff
@@ -11,60 +11,56 @@ class AddAptitudeScoreController extends Controller
 {
     private $applicantModel;
 
-    /**
-     * Instantiates AddApplicantPageController.
-     *
-     * @param ApplicantModel $applicantModel userModel object dependency
-     */
     public function __construct(ApplicantModel $applicantModel)
```
Comment (Harry-Johnston) [inline src/Controllers/API/AddAptitudeScoreController.php:14]: re-add doc block please

File: src/Controllers/API/AddAptitudeScoreController.php [hunk 2]
```diff
@@ -11,60 +11,56 @@ class AddAptitudeScoreController extends Controller
...
             'success' => false,
-            'message' => 'Aptitude score not added',
+            'message' => '',
             'data' => []
         ];
 
-        $statusCode = 500;
-        $aptitudeData = $request->getParsedBody();
+        $matchedApplicantByEmail = $this->applicantModel->getApplicantByEmail($applicantEmail);
+        if ($matchedApplicantByEmail == []) {
```
Comment (Harry-Johnston) [inline src/Controllers/API/AddAptitudeScoreController.php:31]: == or ===?

File: src/Controllers/API/AddAptitudeScoreController.php [hunk 3]
```diff
@@ -11,60 +11,56 @@ class AddAptitudeScoreController extends Controller
...
+        $responseBody = [
             'success' => false,
-            'message' => 'Aptitude score not added',
+            'message' => '',
             'data' => []
         ];
 
-        $statusCode = 500;
-        $aptitudeData = $request->getParsedBody();
+        $matchedApplicantByEmail = $this->applicantModel->getApplicantByEmail($applicantEmail);
```
Comment (Harry-Johnston) [inline src/Controllers/API/AddAptitudeScoreController.php:30]: Is this a semantic name?

File: src/Models/ApplicantModel.php [hunk 4]
```diff
@@ -557,13 +557,39 @@ public function getApplicantByEmail(string $email): array
...
+        ];
+
+        $query = $this->db->prepare($sql);
+        return $query->execute($values);
+    }
+
+    public function submitApplicantAssessmentNotes(int $id, string $assessmentNote): bool
+    {
+        $sql = 'UPDATE `applicants_additional` SET `assessmentNotes` = 
+        CONCAT(COALESCE(`assessmentNotes`, ""), :assessmentNote) WHERE `id` = :id;';
```
Comment (Harry-Johnston) [inline src/Models/ApplicantModel.php:588]: Dont understand the COALESCE but assume you do?

File: src/Models/ApplicantModel.php [hunk 5]
```diff
@@ -557,13 +557,39 @@ public function getApplicantByEmail(string $email): array
...
             'score' => $score,
-            'id' => $id,
+            'id' => $id
+        ];
+
+        $query = $this->db->prepare($sql);
+        return $query->execute($values);
+    }
+
+    public function submitApplicantAssessmentNotes(int $id, string $assessmentNote): bool
```
Comment (Harry-Johnston) [inline src/Models/ApplicantModel.php:585]: Do you need to return a value if you arent using it in the addApptitudeTestScoreController?

**Review by Harry-Johnston — COMMENTED** [review]
Looks great, some minor comments

File: src/Controllers/API/AddAptitudeScoreController.php (hunk 1 above)
Comment (JoshuaBennet) [reply src/Controllers/API/AddAptitudeScoreController.php:14]: done

File: src/Controllers/API/AddAptitudeScoreController.php (hunk 3 above)
Comment (JoshuaBennet) [reply src/Controllers/API/AddAptitudeScoreController.php:30]: $applicantDataByEmail

File: src/Models/ApplicantModel.php (hunk 4 above)
Comment (JoshuaBennet) [reply src/Models/ApplicantModel.php:588]: Good.

File: src/Models/ApplicantModel.php (hunk 5 above)
Comment (JoshuaBennet) [reply src/Models/ApplicantModel.php:585]: We will let Rayna decide.

**Pull request merged**